src/hooks/productData/posthog_ai.tsx

```diff
@@ -87,22 +87,22 @@ export const posthog_ai = {
             ],
             skills: [
                 {
-                    name: 'Traffic watcher',
-                    description: 'Illuminates pathways through clickthrough chaos and millions of pageviews',
+                    name: 'Insight generator',
+                    description: 'AI-generated insights for both event and session-level metrics',
                     sticker: <StickerPath className="size-12" />,
-                    percent: 80,
+                    percent: 30,
                 },
                 {
-                    name: 'UTM untangler',
-                    description: "Makes sense of your team's chaotic campaign naming convention",
+                    name: 'Filter architect',
+                    description: "Not sure what filters to apply? We got you",
```

Comment:
What does this mean? Do we help figure out _what_ to apply as filters or just go through the motions of applying with with natural language? Both the name and description feel a little vague.